package.json

```diff
@@ -7,22 +7,20 @@
   ],
   "scripts": {
     "lerna": "lerna",
-    "start": "lerna run start --stream --parallel",
+    "dev": "turbo run dev --parallel --no-cache",
```

Comment:
Can we keep `start` for the name of the script? if not we should update the documentation at https://github.com/dwarvesf/react-toolkit/blob/master/CONTRIBUTING.md

Reply:
We can keep `start` but then we would need to update the dev script of `docs`. At the moment the `start` script is to run production for `docs` app so we'll have to choose between them.

I think I'll update the CONTRIBUTING.md for now 🤔

Reply:
Sound good to me